DATA/UNIVERSE/SYSTEMS/Bw10/Bw10.ini

```diff
@@ -1195,15 +1294,22 @@ pop_type = gd_im_grp, single_base_law
 relief_time = 20
 density_restriction = 4, unlawfuls
 encounter = area_defend, 19, 0.09
-faction = gd_im_grp, 1
-encounter = area_trade_transport, 13, 0.30
-faction = co_kt_grp, 0.3
-faction = co_nws_grp, 0.3
-faction = co_shi_grp, 0.3
-encounter = area_trade_transport, 19, 0.18
-faction = co_be_grp, 0.2
-faction = co_ti_grp, 0.2
-faction = gd_im_grp, 0.12
+faction = gd_im_grp, 1 ; Independent Miners Guild
+encounter = area_trade_freighter, 13, 0.15

```

Comment:
I think Freelancer generally follows the rule that freighters only appear of a certain faction only within the borders of the colony they are at home at. Otherwise they in borderworlds etc use only trains and transports-

Reply:
Ok. It only made sense because the freighters go into the Tau-31 jumpgate in Leeds.